app/institutions/dashboard/-components/institutional-dashboard-wrapper/styles.scss

```diff
@@ -0,0 +1,57 @@
+@import 'app/styles/layout';
+
+.container {
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+}
+
+.heading-wrapper {
+    border-bottom: 1px solid #ddd;
+}
+
+.banner {
+    padding: 15px 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    div {
+        color: #263947;
+    }
+}
+
+.tab-list {
+    @include clamp-width;
+    white-space: nowrap;
+    position: relative;
+    overflow-x: auto;
+    overflow-y: hidden;
+    box-sizing: border-box;
+    color: rgb(51, 51, 51);
+    line-height: 20px;
+    list-style-image: none;
+    list-style-position: outside;
+    list-style-type: none;
+    height: 41px;
+    padding: 0;
```

Comment:
This is some copy-pasted code from other places we use tabs. Likely better to turn this re-used styling into a mix-in